main.js

```diff
@@ -0,0 +1,22 @@
+let el = document.querySelector("nav ul li");
+    el.remove();
+
+let ele = document.querySelector("p span");
+    ele.remove();
+
+
+
+let lastPost = document.querySelectorAll(".posts article")
+    for(let post of lastPost){
+        if(post.textContent.includes("You -- yes you!")){
+            post.remove();
+        }
+    }
```

Comment:
Interesting way to do this one! I like that it will always target the "You -- yes you!" article no matter its position in the DOM. 
Here's another way to do this if removing the **_last element_** specifically is the main priority:
`querySelectorAll` returns a node list. Node lists are very similar to arrays, so you can grab the last element of a node list in the same way that you'd access the last element of an array: 
`let last = lastPost[lastPost.length -1]`  <-- get last post
`last.remove()`                                          <-- remove it